spec/features/deposit_account_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require 'securerandom'
+
+RSpec.describe Unit::DepositAccount do
+  before do
+    establish_connection_to_api!
+  end
+
+  let(:customer) do
+    Factory.create_individual_customer
+  end
+
+  let(:account) do
```

Comment:
does this create a new customer & account every time `account` is referenced below? I'm seeing 100s of deposit accounts in the Unit sandbox environment, and can't tell if that's just every time the spec is run, or if multiple customers & accounts are being created per spec 🤔 it _might_ be a little cleaner if we did something similar to the ACH payment spec where the create is happening inside of the assertion?
i.e.
```Ruby
it 'creates a deposit account for the customer' do
  account = Unit::DepositAccount.create(deposit_product: 'checking', customer: customer)
  
  expect(account.id).to_not be nil
  etc etc
end
```

Reply:
That's correct & good catch - I went ahead & updated to fix.